megabeast/tools/make_naive_maps.py

```diff
@@ -88,14 +100,18 @@ def create_naive_maps(stats_filename, pix_size=10.0, verbose=False):
                     values = cat[cur_stat + "_" + stat_type][tindxs]
                     values_foreach_pixel[cur_stat][i, j] = values
                     summary_stats[j, i, k] = np.average(values)
-                    summary_sigmas[j, i, k] = np.std(values, ddof=1) / math.sqrt(
-                        len(values)
-                    )
+                    summary_sigmas[j, i, k] = np.std(values, ddof=1) / math.sqrt(len(values))
+                    if median:
+                        summary_stats[j, i, k] = np.median(values)
+                        summary_sigmas[j, i, k] = median_abs_deviation(values)
```

Comment:
What the change to "median_abs_deviation"?  This is different than the np.std. 

In addition, line 103 computes the standard deviation of the mean and "median_abs_deviation" computes the equivalent of the standard deviation.  Feels like the "median_abs_deviation" value needs to be divided by the same number as line 103.

Reply:
Addressed - see updates.